recipes/pybind11_json/all/conanfile.py

```diff
@@ -0,0 +1,47 @@
+from conans import ConanFile, tools
+import os
+
+
+class Pybind11JsonConan(ConanFile):
+    name = "pybind11_json"
+    homepage = "https://github.com/pybind/pybind11_json"
+    description = "An nlohmann_json to pybind11 bridge"
+    topics = (
+        "conan",
+        "header-only",
+        "json",
+        "nlohmann_json",
+        "pybind11",
+        "pybind11_json",
+        "python",
+        "python-binding",
+    )
+    url = "https://github.com/conan-io/conan-center-index"
+    no_copy_source = True
+    license = "BSD-3-Clause"
+
+    @property
+    def _source_subfolder(self):
+        return "source_subfolder"
+
+    def package_id(self):
+        self.info.header_only()
+
+    def requirements(self):
+        self.requires("nlohmann_json/3.9.1")
+        self.requires("pybind11/2.6.2")
+
+    def source(self):
+        tools.get(**self.conan_data["sources"][self.version])
+        extracted_dir = "pybind11_json-" + self.version
+        os.rename(extracted_dir, self._source_subfolder)
+
+    def package(self):
+        self.copy(pattern="LICENSE*", dst="licenses", src=self._source_subfolder)
+        self.copy(
+            "*", dst="include", src=os.path.join(self._source_subfolder, "include")
+        )
+
+    def package_info(self):
+        self.cpp_info.names["cmake_find_package_multi"] = "pybind11_json"
+        self.cpp_info.names["pkg_config"] = "pybind11_json"
```

Comment:
It's still here.
Is this a zombie line that can't be killed :rofl: :wink:

Reply:
I hopefully killed the zombie with adad6da :wink:. Now, we have 

```python
def package_info(self):
    self.cpp_info.names["cmake_find_package"] = "pybind11_json"
    self.cpp_info.names["cmake_find_package_multi"] = "pybind11_json"
```

Is that ok?

Reply:
It's great! thanks!